packages/quantic/force-app/main/default/lwc/quanticCarousel/quanticCarousel.css

```diff
@@ -0,0 +1,43 @@
+.carousel__container {
+  align-items: center;
+}
+
+.carousel__slides {
+  overflow: hidden;
+  flex: 1;
+}
+
+.carousel__indicators-container {
+  justify-content: center;
+}
+
+.carousel__indicator {
+  width: 3.25rem;
+  height: 0.25rem;
+  border: 0;
+  border-radius: var(--lwc-borderRadiusMedium, 0.25rem);
```

Comment:
`rem` is relative to the font-size of the root `html` tag (customizable in the browser settings).

I know that Tailwind uses `rem` for the border radius by default (see https://tailwindcss.com/docs/border-radius). I don't feel strongly either way.

Reply:
Used `rem` because SLDS uses this unit as well when it comes to border radius:
<img width="1059" alt="Radius" src="https://github.com/coveo/ui-kit/assets/86681870/01b8a68a-be5c-4f53-9e23-aa8c4bd90d61">

Reply:
Oh yea, I had the two mixed up... 🤦‍♂️